src/scales/scale.logarithmic.js

```diff
@@ -174,8 +174,8 @@ module.exports = Scale.extend({
 		var DEFAULT_MIN = 1;
 		var DEFAULT_MAX = 10;
 
-		me.min = valueOrDefault(tickOpts.min, me.min);
-		me.max = valueOrDefault(tickOpts.max, me.max);
+		me.min = tickOpts.min >= 0 ? tickOpts.min : me.min;
```

Comment:
What happens if min/max are `''` or `false`

Reply:
good catch. fixed

Reply:
Do we support `log(0)` (-infinity)? if not, I think it's better to do:

```javascript
me.min = tickOpts.min > 0 ? tickOpts.min : me.min;
me.max = tickOpts.max > 0 ? tickOpts.max : me.max;
```

Reply:
Doing that causes some tests to fail. It looks like we already have special cases for `0` where we use `DEFAULT_MIN` and `DEFAULT_MAX` in that case. It's sort of a weird behavior though. If you set `max = foobar` then I'm not sure why we'd change it to `max = 10`. It feels arbitrary and more intuitive to just ignore it and let max be the max value of the data